src/gui/widgets/Knob.cpp

```diff
@@ -172,6 +174,29 @@ void Knob::setLabel( const QString & txt )
 					horizontalAdvance(QFontMetrics(pointSizeF(font(), 6.5)), m_label)),
 						m_knobPixmap->height() + 10);
 	}
+
+	update();
+}
+
+
+void Knob::setHtmlLabel(const QString &htmltxt)
+{
+	m_label = htmltxt;
+	m_isHtmlLabel = true;
```

Comment:
Should `setLabel` set `m_isHtmlLabel = false;` too? That way the label can be changed freely.